elevenlabs_api_benchmark.py

```diff
@@ -68,10 +77,12 @@
 general_group = parser.add_argument_group('General Parameters')
 general_group.add_argument("--voice_id", default=DEFAULT_VOICE_ID,
                            help="ID of the voice for speech synthesis")
+general_group.add_argument("--detect_audio_format", action='store_true',
```

Comment:
use all dashes in args, i.e., "--detect-audio-format". But maybe call this --analyze-audio or --analyze-audio-format?